### DISCUSSION TITLE
Blog post not showing up

### DISCUSSION THREAD

**Discussion by ammaryasirnaich** [academicpages/academicpages.github.io#2019] [Q&A]
Hi, I am having an issue making the blog post work on the page. After following the template setting for new posts under **_post** directory and pushing it to GitHub pages, it doesn't seem to show the page and gives an error page not found. I even can't access the default post that come with the template. 
Would appreciate the help from anyone thanks

**Comment by rjzupkoii** [comment]
Can you link to the repository that you are working with? It's hard to say what the issue might be without a closer look at the repository.

**Reply by ammaryasirnaich** [reply]
https://github.com/ammaryasirnaich/ammaryasirnaich.github.io  

Above is the link

**Reply by rjzupkoii** [reply]
@ammaryasirnaich Try undoing the change that was made in [this commit](https://github.com/ammaryasirnaich/ammaryasirnaich.github.io/commit/1e8bdeecf9e945ce719c66ae74f4ea6c52d72c99). That's the first thing that stands out to me in the changes to your repository.

**Reply by ammaryasirnaich** [reply]
@rjzupkoii, i have reverted the changes that i had made in _config.yml but the issue remains the same 

![Selection_069](https://github.com/academicpages/academicpages.github.io/assets/5053086/9172d7fc-6015-446b-b150-564debed434a)

**Reply by rjzupkoii** [reply]
@ammaryasirnaich So it seems like something is going on with the configuration since [this URL](https://ammaryasirnaich.github.io/year-archive/posts/) sends me to the custom 404 page while the [blog post entry URL](https://ammaryasirnaich.github.io/posts/2012/08/blog-post-4/) sends me to the GitHub 404 page. You might want to try swapping out your `_config.yml` for [the default one for the template](https://github.com/academicpages/academicpages.github.io/blob/master/_config.yml) and seeing if you can navigate to the posts again.

**Reply by ammaryasirnaich** [reply]
@rjzupkoii After swapping it with the default config is **now redirecting to the correct path**. I have created a new 2024-03-02-testing-post.md file in **_post** how can I make it appear in the same blog post category?

**Reply by rjzupkoii** [reply]
@ammaryasirnaich There is an issue on the backlog for adding new categories / sections (#1655) and it's fairly high on the list of things that I'm going to be working on. I suspect that the functionality is already there but even then I'll likely have to write-up an new wiki page to walk people through out to do it.

**Reply by ammaryasirnaich** [reply]
@rjzupkoii, thanks for the heads-up on this issue really appreciate your time to figure it out for me. I will be looking into it as it's resolved.
